src/CoreShop/Bundle/OrderBundle/Pimcore/Repository/CartRepository.php

```diff
@@ -73,6 +75,7 @@ public function findCartById($id): ?OrderInterface
     {
         $list = $this->getList();
         $list->setCondition('o_id = ? AND order__id is null ', [$id]);
+        $list->setLimit(1);
```

Comment:
that should always ever return one or null since we directly filter on o_id.

Reply:
yes, but as far as i know, the query is faster with a limit, even if you won't notice it in reality :)

Reply:
I am not sure, it feels wrong here, so please remove it